### ISSUE TITLE
Interpolation of aparcaseg registered to T1w

### ISSUE THREAD

**Issue by kaitj** [nipreps/fmriprep#2580]
The registered segmentations to subject T1w (`sub-{subject}_space-T1w_desc-aparcaseg_dseg.nii.gz`) appear to be incorrectly interpolating the labels. Screenshot of example included at bottom of bug report. Unclear if this affects other transformed segmentations. Original segmentations are correctly labelled.

### What version of fMRIPrep are you using?
21.0.0rc1; also replicated in 20.2.1 
(issue seems to begin to appear around 20.x as far as I can tell, transformed segmentations were fine in 1.5.x)

### What kind of installation are you using? Containers (Singularity, Docker), or "bare-metal"?
Singularity (invoked via Snakemake)

### What is the exact command-line you used?
(pulled from Snakemake's Snakefile)
```
'singularity run -e {params.fmriprep_sif} '
' ${{SLURM_TMPDIR}}/bids_{wildcards.subject} ${{SLURM_TMPDIR}}/fmriprep_{wildcards.subject} participant '
' --participant_label {wildcards.subject} '
' --fs-license-file {params.fs_license} '
' -w ${{SLURM_TMPDIR}}/fmriprep_work_{wildcards.subject} '
' --n_cpus {threads} '
' --omp-nthreads {threads} '
' --output-spaces MNI152NLin6Asym MNI152NLin2009cAsym T1w '
' --cifti-output '
' --use-aroma '
' --output-layout bids '
' --notrack  && '
' cp -Rv ${{SLURM_TMPDIR}}/fmriprep_{wildcards.subject}/sub-{wildcards.subject} {output.fmriprep_out}'
```

### Have you checked that your inputs are BIDS valid?
Yes

### Did fMRIPrep generate the visual report for this particular subject? If yes, could you share it?
Yes (although the segmentations are not included, can still have include a visual report if needed)

### Can you find some traces of the error reported in the visual report (at the bottom) or in *crashfiles*?
No

### Are you reusing previously computed results (e.g., FreeSurfer, Anatomical derivatives, work directory of previous run)?
FreeSurfer 7.1 (ran separately). Also replicated with a clean run.

### fMRIPrep log
If you have access to the output logged by fMRIPrep, please make sure to attach it as a text file to this issue.

List the steps you performed that revealed the bug to you.
Include any code samples. Enclose them in triple back-ticks (```)
Like this:
1. Opened subject T1w and `sub-{subject}_space-T1w_desc-aparcaseg_dseg.nii.gz` in FSLeyes
2. Changed color table from greyscale to FS cortical (also inspecting the labels for a few voxels)

![image](https://user-images.githubusercontent.com/21226986/136393892-456d1955-b0c3-4ff5-9efc-22d1a6817a99.png)

**Comment by effigies** [comment]
Thanks for reporting this. Looks like I broke things in nipreps/smriprep#201. We'll need to release a fixed version of sMRIPrep for both the 0.7.x series and 0.8.x series and then we can release fixed versions of fMRIPrep.

**Comment by effigies** [comment]
20.2.5 is released.
